docs/testplans/ras/ras-ingestion-test-plan.md

```diff
@@ -0,0 +1,41 @@
+# Test plan for Gen3's implementation of Researcher Auth Services (RAS)
+
+## One-liner overview of the service
+The Resercher Auth Services (RAS) is an implementation of the Global Alliance for Genomics & Health (GA4GH) Passport Specifications ([https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md)). It defines a common process for Authentication, Authorization and Logging around NIH's open and controlled data assets and repositories. This test plan focuses on the Gen3 implementation of such spec.
+More info:  ([https://auth.nih.gov/docs/RAS/serviceofferings.html](https://auth.nih.gov/docs/RAS/serviceofferings.html))
+
+## Environmental dependencies
+In order to execute this test, the environment must have the following items fully configured:
+- Latest fence version with the RAS HTTP API endpoints
```

Comment:
So initially we'll be rolling this out as a branch release of fence _only_ on staging environments. This is because RAS is not production ready yet. 

Once RAS is production ready, I think this statement will be true because we'd fold it into a monthly release for Fence